_build/maintainer_guide.rst

```diff
@@ -0,0 +1,293 @@
+Symfony Docs Maintainer Guide
+=============================
+
+The `symfony/symfony-docs`_ repository stores the Symfony project documentation
+and it's managed by the `Symfony Docs team`_. This article explains in detail
```

Comment:
I think _it's_ should be replaced by _is_. I think the readability of the sentence will improve.

```suggestion
and is managed by the `Symfony Docs team`_. This article explains in detail
```

Reply:
Thanks for your correction! You are right! Instead of accepting your suggested change, I've done it myself. There's nothing wrong with your suggestion ... but GitHub creates commits of different people and our "gh" tool can't squash them. I'd like Oskar to merge this PR, so I prefer to have commits from only one person to allow automatic squashing. Thanks for understanding!